src/components/TodoContainer.js

```diff
@@ -1,19 +1,20 @@
-import React, { useState, useEffect, useCallback } from 'react';
-import TodoList from './TodoList';
-import AddTodoForm from './AddTodoForm';
-import TodoListToggle from './TodoListToggle';
-import api from './Api';
-import style from './App.css';
-import PropTypes from 'prop-types';
+import React, { useState, useEffect, useCallback } from "react";
+import TodoList from "./TodoList";
+import AddTodoForm from "./AddTodoForm";
+import TodoListToggle from "./TodoListToggle";
+import api from "./Api";
+import style from "./App.css";
+import PropTypes from "prop-types";
+import TodoNavigation from "./TodoNavigation";
 
 const TodoContainer = ({ tableName }) => {
   const [todoList, setTodoList] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
-  const [checkedState, setCheckedState] = useState({});
+  const [isChecked, setIsChecked] = useState([]);
```

Comment:
the whole checkbox system is having some problems right now, and I can't tell if you were trying to temporarily remove the feature, or now you're adding it back in as a state-only bit but going to add later on the ability to modify the backend

take a look at the errors when you check a `TodoListItem` and I think we should decide to fix that up so it at least works with the state, or maybe disable some of it for now